semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -186,6 +189,61 @@ def to_markdown_attr(
             )
         )
 
+    def to_markdown_attribute_table(
+        self, semconv: BaseSemanticConvention, output: io.StringIO
+    ):
+        attr_to_print = []
+        for attr in sorted(
+            semconv.attributes, key=lambda a: "" if a.ref is None else a.ref
+        ):
+            if self.render_ctx.group_key is not None:
+                if attr.tag == self.render_ctx.group_key:
+                    attr_to_print.append(attr)
+                continue
+            if self.render_ctx.is_full or attr.is_local:
+                attr_to_print.append(attr)
+        if self.render_ctx.group_key is not None and not attr_to_print:
+            raise ValueError(
+                "No attributes retained for '{}' filtering by '{}'".format(
+                    semconv.semconv_id, self.render_ctx.group_key
+                )
+            )
+        if attr_to_print:
+            output.write(MarkdownRenderer.table_headers)
+            for attr in attr_to_print:
+                self.to_markdown_attr(attr, output)
+        attr_sampling_relevant = [
+            attr for attr in attr_to_print if attr.sampling_relevant
+        ]
+        self.to_creation_time_attributes(attr_sampling_relevant, output)
+
+    @staticmethod
+    def to_markdown_metric_table(
+        semconv: MetricSemanticConvention, output: io.StringIO
+    ):
+        """
+        This method renders metrics as markdown table entry
+        """
+        if not isinstance(semconv, MetricSemanticConvention):
+            raise ValueError(
+                "semconv `{}` was specified with `metric_table`, but it is not a metric convention".format(
+                    semconv.semconv_id
+                )
+            )
+
+        output.write(
+            "| Name     | Instrument Type | Unit (UCUM) | Description    |\n"
```

Comment:
For me, we should pick one. Having to define the unit twice in the YAML seems superfluous. I feel like `By` is pretty obvious, particularly when paired with the metric description. Also, any non-UCUM units like `{requests}` are also pretty obvious. 

Perhaps if we want to add this later on, it can be in another PR?

Reply:
Fair enough, let's go with this for now!